dotcom-rendering/src/web/browser/ophan/index.ts

```diff
@@ -8,6 +8,7 @@ export const ophan = (): Promise<void> => {
 	record({ edition: window.guardian.config.page.edition });
 
 	// Record server-side AB test variants (i.e. control or variant)
+	// eslint-disable-next-line @typescript-eslint/strict-boolean-expressions, @typescript-eslint/no-unnecessary-condition -- Client-side safety,
```

Comment:
Should we make it optional in our type definitions?

Reply:
Looking at this again I don't think we gain any safety ... updated